src/core/MouseCapture.tsx

```diff
@@ -102,53 +131,52 @@ export default class MouseCapture extends React.Component<Props, State> {
         const { xScale, yScale } = this._createPhysicalToLogicalScales();
         this.props.onDragStart(xScale(e.clientX), yScale(e.clientY), e);
       }
+
+      this._removeWindowMouseEventHandlers();
+      this._addWindowMouseEventHandlers();
     }
   };
 
-  private _maybeDispatchDragHandler(e: React.MouseEvent, handler: (xPct: number, yPct: number, e: React.MouseEvent) => void) {
-    if (e.button === LEFT_MOUSE_BUTTON && handler && this.state.mouseDownClientX != null) {
+  private _onMouseMoveInCaptureArea = (e: React.MouseEvent) => {
+    if (this.props.onHover) {
       const { xScale, yScale } = this._createPhysicalToLogicalScales();
-      handler(
-        xScale(e.clientX),
-        yScale(e.clientY),
-        e
-      );
-    }
-  }
+      this.props.onHover(xScale(e.clientX), yScale(e.clientY), e);
 
-  private _onMouseMove = (e: React.MouseEvent) => {
-    this._maybeDispatchDragHandler(e, this.props.onDrag);
+      // If onHover exists, event.stopPropagation is called and the window event does not get called.
```

Comment:
This is somewhat awkward, since nothing in _this_ component calls `stopPropagation`, by design. Unfortunately this is not a safe assumption to make, since I know there are already other components that rely on this one and don't necessarily stop propagation. I wish I remembered why I added that behavior to InteractionCaptureLayer -- the only thing I can think of off the top of my head is that, because the component is very large, almost all mousedown-mouseup interactions will trigger a meaningless click, which is why this component has its own definition of "click" as defined by how far the mouse moved. Have you tried seeing what happens if you remove those `stopPropagation`s? I always considered that function a bit of a hack.

This comment applies to all of the explicit calls to the `window` methods.